go.mod

```diff
@@ -1 +1,3 @@
 module github.com/unidoc/unioffice
+
+go 1.12
```

Comment:
Please remove the go.mod file.  Not necessary and this is expected to work on older go versions.